test/integration/test_integration_basics.py

```diff
@@ -441,3 +442,67 @@ def test_bad_arg_error_messages(http_test_server_fixture):
       expect_failure=True,
       as_json=False)
   assert "Bad argument: Termination predicate 'a:a' has an out of range threshold." in err
+
+
+def test_multiple_backends_http_h1(multi_http_test_server_fixture):
+  """
+  Runs the CLI configured to use plain HTTP/1 against multiple test servers, and sanity
```

Comment:
Could we also verify we spread requests as intended across individual back-ends? (So the avg number of replies observed from a backend / received by a backend looks as expected?)

Reply:
Done

Reply:
Done